packages/design-system/src/styles/components/_Table.scss

```diff
@@ -40,13 +41,9 @@
   }
 }
 
-.ds-c-table__header--label span {
-  display: block;
-  text-align: center;
-  text-transform: uppercase;
-}
-
 @mixin ds-c-table--stacked {
+  $stacked-title-max-width: 160px;
```

Comment:
I think we might want to change this variable name to be more specific - what about `$table-stacked-title-max-width`?

Reply:
@line47 I think this has been addressed, and we went with `$table-stacked-header-max-width` to be consistent with the header terminology